cmd/manager/main.go

```diff
@@ -192,6 +193,11 @@ func Command() *cobra.Command {
 		"",
 		"Set the IP family to use. Possible values: IPv4, IPv6, \"\" (= auto-detect) ",
 	)
+	cmd.Flags().Duration(
+		operator.KubeClientTimeout,
+		60*time.Second,
```

Comment:
Am I right that this default of 60 seconds changes the current behaviour for some usages (discovery has 32s apparently)? Don't think it is a big issue, though.

Reply:
Going by the [comment](https://github.com/kubernetes/client-go/blob/becbabb360023e1825a48b4db85f454e452ae249/discovery/discovery_client.go#L48-L49) in `DiscoveryClient` it looks like the 32s timeout is an arbitrary value used to distinguish that particular instance from among many different timeouts. It is only used when the REST client doesn't have its own timeout set (which we are setting now with this change). I don't think it will have a serious effect, except maybe taking twice as long to detect a problem with discovery. 

Of course, there may be other unforeseen side effects from changing the global timeout. I am not too worried about that because we just need to set `--kube-client-timeout=0` to get back to the old behaviour.